NextCenturyCorporation/itm-ingest
# Itm 519

[ITM-519](https://nextcentury.atlassian.net/jira/software/projects/ITM/boards/116?assignee=60ba425da547eb00686ee0ce&selectedIssue=ITM-519)

To test:

- delete survey version 3.0 from your `delegationConfig` collection (if you have one)
- enter `/delegation-survey` directory
- run `python3 convert_adms_for_delegation.py`
- run `python3 update_survey_config.py` and press `Enter` and `2` 

You should then see the updated survey version 3.0 in the `delegationConfig` collection.

Then view the [dashboard PR ](https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/compare/ITM-519?expand=1)

[ITM-519]: https://nextcentury.atlassian.net/browse/ITM-519?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

EDIT: I just added a new script that makes setup easier. From inside `/delegation-survey` now you just need to run `python3 survey_setup_script.py`
